### DISCUSSION TITLE
BDMV播放设置问题

### DISCUSSION THREAD

**Discussion by Swoon0331** [dyphire/mpv-config#28] [Q&A]
感谢兄弟贡献了如此优秀的修改版MPV，在使用功能上带来了很大的便利。只是个人碍于刚从Potplayer转过来诸多设置仍在摸索，相关播放问题还望不吝解答：下载的其它一些版本的MPV能直接播放BDMV，但这个改版试了几个片子都似乎不行，是需要修改哪里参数否？先谢过了！

**Comment by dyphire** [comment]
下了个BDMV测试了下，无法复现。无论是直接拖放BDMV文件夹、直接播放MovieObject.bdmv还是直接播放m2ts文件都可以正常识别并播放
请提供具体的播放步骤和日志文件

**Comment by Hyphen-H** [comment]
自己remux成mkv呗。。。DGdemux，makemkv都是傻瓜操作。。要么eac3to+mkvtoolnix

**Comment by dyphire** [comment]
鉴于无法复现且长期未得到回应，暂且关闭此问题
